tt_metal/impl/dispatch/memcpy.hpp

```diff
@@ -24,83 +45,98 @@ namespace tt::tt_metal {
 #if defined(__x86_64__) || defined(__i386__)
 template <bool debug_sync = false>
 static inline void memcpy_to_device(void* __restrict dst, const void* __restrict src, size_t n) {
+    // Ensure destination is properly aligned for optimal SIMD performance
     TT_ASSERT((uintptr_t)dst % MEMCPY_ALIGNMENT == 0);
 
+    // Configuration for bulk processing: inner loop processes 8 x 32-byte operations
+    // This creates 256-byte blocks (8 * 32 = 256 bytes) for maximum throughput
     static constexpr uint32_t inner_loop = 8;
-    static constexpr uint32_t inner_blk_size = inner_loop * sizeof(__m256i);
+    static constexpr uint32_t inner_blk_size = inner_loop * sizeof(__m256i);  // 256 bytes
```

Comment:
Please fix this and the variable above as well:
```suggestion
    constexpr uint32_t inner_blk_size = inner_loop * sizeof(__m256i);  // 256 bytes
```
`static` here means "static storage duration" which is unnecessary because it doesn't need to exist as a runtime constant, only compile-time.